<?xml version="1.0" encoding="utf-8"?><testsuite errors="0" failures="1" name="pytest" skips="0" tests="3" time="0.293"><testcase classname="test.test_formater.TestFormater" file="test/test_formater.py" line="5" name="test_plain_lowercase" time="0.000574111938477"></testcase><testcase classname="test.test_views.FlaskrTestCase" file="test/test_views.py" line="14" name="test_msg_with_output" time="0.0249600410461"><failure message="AssertionError: &apos;{ &quot;imie&quot;:&quot;Lukasz&quot;, &quot;mgs&quot;:Hello World!&quot;}&apos; != &apos;&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 3.2 Final//EN&quot;&gt;\n&lt;title&gt;404 Not Found&lt;/title&gt;\n&lt;h1&gt;Not Found&lt;/h1&gt;\n&lt;p&gt;The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.&lt;/p&gt;\n&apos;">self = &lt;test_views.FlaskrTestCase testMethod=test_msg_with_output&gt;

    def test_msg_with_output(self):
        rv = self.app.get(&apos;/?output=json&apos;)
&gt;       self.assertEquals(&apos;{ &quot;imie&quot;:&quot;Lukasz&quot;, &quot;mgs&quot;:Hello World!&quot;}&apos;, rv.data)
E       AssertionError: &apos;{ &quot;imie&quot;:&quot;Lukasz&quot;, &quot;mgs&quot;:Hello World!&quot;}&apos; != &apos;&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 3.2 Final//EN&quot;&gt;\n&lt;title&gt;404 Not Found&lt;/title&gt;\n&lt;h1&gt;Not Found&lt;/h1&gt;\n&lt;p&gt;The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.&lt;/p&gt;\n&apos;

test/test_views.py:17: AssertionError</failure></testcase><testcase classname="test.test_views.FlaskrTestCase" file="test/test_views.py" line="10" name="test_outputs" time="0.00268793106079"></testcase></testsuite>